packages/ckeditor5-block-quote/src/blockquoteui.ts

```diff
@@ -33,29 +33,51 @@ export default class BlockQuoteUI extends Plugin {
 	 */
 	public init(): void {
 		const editor = this.editor;
-		const t = editor.t;
+		const command: BlockQuoteCommand = editor.commands.get( 'blockQuote' )!;
```

Comment:
This change might require adding `requires` getter to ensure that the feature's editing part is loaded before the UI part. @oleq